webpack.demo.js

```diff
@@ -0,0 +1,54 @@
+var webpack = require('webpack');
+var HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: {
+    'angular2': [
+      'zone.js',
+      'reflect-metadata',
```

Comment:
We now have https://code.angularjs.org/2.0.0-alpha.48/external-dependencies.js bundle that contains all the external dependencies (zone and reflect-metadata for now) so we can use it (and by this I mean just stick a good old script tag) instead of having it done through webpack